vespa-dependencies-enforcer/allowed-maven-dependencies.txt

```diff
@@ -128,10 +128,10 @@ org.apache.opennlp:opennlp-tools:${opennlp.vespa.version}
 org.apache.velocity:velocity-engine-core:${velocity.vespa.version}
 org.apache.yetus:audience-annotations:0.12.0
 org.apache.zookeeper:zookeeper-jute:${zookeeper.client.vespa.version}
-org.apache.zookeeper:zookeeper-jute:3.8.1
+org.apache.zookeeper:zookeeper-jute:3.8.0
```

Comment:
`zookeeper.client.vespa.version` == 3.8.0?

Reply:
Ah, yes, probably as issue from me mixing up branches a bit.

Reply:
... or it's the dependency enforcer that doesn't identify the two?

Reply:
(It accepts this list, so I suggest we do too, and it'll be a single version once this version bump is complete.)